cdbot/constants.py

```diff
@@ -140,7 +144,7 @@ class Exchange:
     "https://pbs.twimg.com/profile_images/921313066515615745/fLEl2Gfa_400x400.jpg"
 )
 ELITECOUNT_ENABLED = True
-
+GAMECOUNT_ENABLED = True
```

Comment:
Tbh we should do stuff like this with the database this should be changeable with a command. But its probs fine. Cd dead anyway

Reply:
No we shouldn't do this with the database, none of CD bot is setup for feature flags to live in a database